website/docs/advanced/proxy/overview.md

```diff
@@ -0,0 +1,1258 @@
+<TabItem value="env-vars" label="Environment variable">
+
+```shell
+CONFIGCAT_MY_SDK_OFFLINE_LOCAL_POLL_INTERVAL=5
+```
+
+</TabItem>
+</Tabs>
+
+</td>
+<td><code>5</code></td>
+<td>The polling interval of the local file in seconds.</td>
+</tr>
+</tbody>
+</table>
+
+### Cache
+
+You have the option to cache feature flag evaluation data in an external cache. Currently, the only available option is <a target="blank" href="https://redis.io">Redis</a>.  
+The cache key for the underlying SDKs is based on the [SDK Key](#sdk-identifier--sdk-key). This means that multiple Proxy instances using the same SDK key will read/write the same cache entry.
+
+:::info
+The ConfigCat Proxy supports *shared caching*. This means it can feed an external cache that is shared by other ConfigCat SDKs. You can read more about this feature and the required minimum SDK versions [here](/docs/advanced/caching/#shared-cache).
+
+<img className="bordered zoomable" src="/docs/assets/proxy/shared_cache.png" alt="Shared cache architecture" />
+
+:::
+
+<Tabs groupId="yaml-env">
+<TabItem value="yaml" label="YAML" default>
+
+This is the whole YAML section of the cache specific options.
+```yaml title="options.yml"
+cache:
+  redis:
+    enabled: <true|false>
+    db: 0
+    user: "<user>"
+    password: "<pass>"
+    addresses: ["<addr1>", "<addr2>"]
+    tls: 
+      enabled: <true|false>
+      min_version: <1.0|1.1|1.2|1.3>
+      server_name: "<server-name>"
+      certificates:
+        - cert: "<path-to-cert>"
+          key: "<path-to-key>"
+```
+
+</TabItem>
+<TabItem value="env-vars" label="Environment variables">
+
+These are the cache specific environment variables.
+```shell
+CONFIGCAT_CACHE_REDIS_ENABLED=<true|false>
+CONFIGCAT_CACHE_REDIS_DB=0
+CONFIGCAT_CACHE_REDIS_USER="<user>"
+CONFIGCAT_CACHE_REDIS_PASSWORD="<pass>"
+CONFIGCAT_CACHE_REDIS_ADDRESSES='["<addr1>", "<addr2>"]'
+CONFIGCAT_CACHE_REDIS_TLS_ENABLED=<true|false>
+CONFIGCAT_CACHE_REDIS_TLS_MIN_VERSION=<1.0|1.1|1.2|1.3>
+CONFIGCAT_CACHE_REDIS_TLS_SERVER_NAME="<server-name>"
+CONFIGCAT_CACHE_REDIS_TLS_CERTIFICATES='[{"key":"<path-to-key>","cert":"<path-to-cert>"}]'
+```
+
+</TabItem>
+</Tabs>
+
+Here's the explanation for each option:
+
+#### Redis
+
+<table className="proxy-arg-table">
+<thead><tr><th>Option</th><th>Default</th><th>Description</th></tr></thead>
+<tbody>
+
+<tr>
+<td>
+
+<Tabs groupId="yaml-env">
+<TabItem value="yaml" label="YAML" default>
+
+```yaml
+cache:
+  redis:
+    addresses: ["<addr1>", "<addr2>"]
+```
+
+</TabItem>
+<TabItem value="env-vars" label="Environment variable">
+
+```shell
+CONFIGCAT_CACHE_REDIS_ADDRESSES='["<addr1>", "<addr2>"]'
+```
+
+</TabItem>
+</Tabs>
+
+</td>
+<td><code>["localhost:6379"]</code></td>
+<td>The addresses of the Redis instances. The Proxy uses <a target="blank" href="https://redis.uptrace.dev/guide/universal.html">Universal Redis clients</a>, so if the array contains multiple addresses it will use <code>ClusterClient</code> instances.</td>
```

Comment:
```suggestion
<td>The addresses of the Redis instances. The Proxy uses <a target="blank" href="https://redis.uptrace.dev/guide/universal.html">Universal Redis clients</a>, so if the array contains multiple addresses, it will use <code>ClusterClient</code> instances.</td>
```